src/screens/user/user.js

```diff
@@ -370,25 +411,25 @@ export default class UsersScreen {
     const propsForCards = createPropsForCards(userOrders);
 
     propsForCards.forEach((props) => {
-      const card = new Card(target.querySelector('.menus-cards-container'), props);
-      const containerForCard = document.createElement('div');
+      const cardContainer = document.createElement('div');
 
-      target.querySelector('.menus-cards-container').appendChild(containerForCard);
-      card.render(containerForCard, props);
+      target.querySelector('.menus-cards-container').appendChild(cardContainer);
+      const card = new Card(cardContainer, props);
+      card.render(cardContainer, props);
 
       this.cards.push(card);
     });
 
     return screen;
   }
-  /*
-    update(cardUpdates) {
-      const date = new Date(cardUpdates.header.date);
-      for (const card of this.cards) {
-        if (new Date(card.header.date) === date) {
-          card.render(cardUpdates);
-        }
+
+  update(cardUpdates) {
+    const date = new Date(cardUpdates.header.date);
+    for (const card of this.cards) {
+      if (new Date(card.props.header.date).getTime() === date.getTime()) {
```

Comment:
Instead of card.props.header.date I would use card.key or card.id, where id and key store the unique identified of the card component.